core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-page-tools-seo/dot-page-tools-seo.component.scss

```diff
@@ -53,16 +57,27 @@
 
 ::ng-deep {
     .page-tools-dialog {
-        .p-dialog-right {
+        &.p-dialog-right {
             .p-dialog {
                 margin-right: $spacing-9;
             }
         }
+
         .p-dialog {
             .p-dialog-header {
                 padding: $spacing-4;
                 border-bottom: 1px solid $color-palette-gray-300;
+
+                .p-dialog-header-icon {
+                    display: none;
```

Comment:
not sure, but maybe we want to do `opacity: 0` instead of this. What do you think? @fmontes